inji-web/src/__tests__/components/Common/AppToaster.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {  screen } from '@testing-library/react';
+import { AppToaster } from '../../../components/Common/AppToaster';
+import { reduxStore } from '../../../redux/reduxStore'; // Assuming the store is named reduxStore
+import { renderWithProvider } from '../../../test-utils/mockUtils';
+
+describe('AppToaster Layout Tests', () => {
+    test('renders correctly with English language', () => {
+        const { asFragment } = renderWithProvider(<AppToaster />);
+        reduxStore.dispatch({ type: 'SET_LANGUAGE', payload: 'en' });
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    test('renders correctly with Arabic language', () => {
+        const { asFragment } = renderWithProvider(<AppToaster />);
+        reduxStore.dispatch({ type: 'SET_LANGUAGE', payload: 'ar' });
+        expect(asFragment()).toMatchSnapshot();
+    });
+});
+
+describe('AppToaster Behavioral and Functionality Tests', () => {
+    test('checks if renders correctly with English language', () => {
```

Comment:
In all the test files for the all the individual test I have used term "Check" because using the same prefix as the block name is not being advised by eslint